### DISCUSSION TITLE
discuss: fleek failures

### DISCUSSION THREAD

**Discussion by bhargavaparoksham** [gitcoinco/grants-stack#621] [Q&A]
Scenario:

The protocol's frontend is hosted on fleek. If the service goes down -> we should be able to relay that information to the users & also have a backup.

Short term solution:

If fleek goes down replace dns to directly hit ipfs link.

**Comment by thelostone-mc** [comment]
- Add in uptime monitoring
- Take snapshot of releases 
- Deploy onto another service (central hosted)
- Update S3 router -> to point to central service instead of fleek

**Comment by vacekj** [comment]
This can be solved by using a service such as Vercel to quickly deploy the frontends and change DNS. We should stage a drill of this.

**Reply by codenamejason** [reply]
I like this idea

**Comment by vacekj** [comment]
Solved by our ongoing migration to Vercel.
